documents/specs/Elements.txt

```diff
@@ -536,6 +536,7 @@ k∩                                   = "aeiouyAEIOUY"
 kṗ                                   = phi
 k⋏                                   = 2 ** 20
 k⋎                                   = 2 ** 30
+k*                                   = "+-<>[].,"
```

Comment:
If we're going to have this, we should add *everything* from #160 .

Reply:
I think they should be added separately and one at a time. Arguably this shouldn't be added now at all

Reply:
AUsername means adding them to the element list, rather than implementing them.

Reply:
IMO it doesn't make much sense to add things to the element list separately; they should be added in the same PR as the code that implements them

Reply:
Sure, but I added some of the requests from #160 to the elements list before this, so they suggested to go the full way.